generator-cas-app/generators/app/index.js

```diff
@@ -0,0 +1,65 @@
+"use strict";
+const Generator = require("yeoman-generator");
+const chalk = require("chalk");
+const yosay = require("yosay");
+
+module.exports = class extends Generator {
+  async prompting() {
+    this.log(
+      yosay(
+        `Welcome to the ${chalk.red("generator-cas-app")} generator!
+        This generator will set up:
+          1) Your asdf package manager https://asdf-vm.com/ .tool-versions file
+          2) Your requirements.txt file
+          3) Your Makefile`
+      )
+    );
+
+    this.answers = await this.prompt([
+      {
+        type: "input",
+        name: "nodejs",
+        message: "node.js version",
+        default: "14.17.6"
+      },
+      {
+        type: "input",
+        name: "yarn",
+        message: "yarn version",
+        default: "1.22.5"
```

Comment:
This should probably default to `1.22.10`
Something we discovered while starting a new project at Button is that the asdf yarn plugin only supports yarn 1. Yarn 2 or 3 are managed differently. Not sure if this is something we want to handle. If we wanted to support yarn 2+, the solution is to have `yarn 1.22.10` in the `.tool-versions` and use the `yarn set version` command (https://yarnpkg.com/cli/set/version)